src/components/collections/button.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from "@emotion/styled";
+
+interface ButtonProps {
+    label: string;
+    click: () => void;
+    color: string;
+    borderRadius: string;
+}
+
+interface ButtonStyleProps {
+    variantcolor: string;
+    borderRadius: string;
+}
+
+// const ButtonUi = styled.button`
+//     background-color: ${(props: ButtonStyleProps) =>
+//         props.variantcolor ? props.variantcolor : "gray"};
+//     border: none;
+//     border-radius: ${(props: ButtonStyleProps) => props.borderRadius};
+//     color: white;
+//     padding: 10px 15px;borderRadius
+//     text-align: center;
+//     text-decoration: none;
+//     display: inline-block;
+//     font-size: 10px;
+// `;
```

Comment:
please remove the commented code

```suggestion
// const ButtonUi = styled.button`
//     background-color: ${(props: ButtonStyleProps) =>
//         props.variantcolor ? props.variantcolor : "gray"};
//     border: none;
//     border-radius: ${(props: ButtonStyleProps) => props.borderRadius};
//     color: white;
//     padding: 10px 15px;borderRadius
//     text-align: center;
//     text-decoration: none;
//     display: inline-block;
//     font-size: 10px;
// `;
```